code/game/machinery/autolathe.dm

```diff
@@ -97,9 +98,11 @@
 	viewing_mobs -= user
 
 /obj/machinery/autolathe/ui_static_data(mob/user)
-	var/list/data = list()
+	var/list/data = list()	
 	data["acceptsDisk"] = TRUE
 
+	var/list/ore_values = list(iron = 1, glass = 1, copper = 5, plasma = 15, silver = 16, gold = 18, titanium = 30, uranium = 30, diamond = 50, bluespace = 50, bananium = 60)
```

Comment:
```suggestion
	var/static/list/ore_values = list(iron = 1, glass = 1, copper = 5, plasma = 15, silver = 16, gold = 18, titanium = 30, uranium = 30, diamond = 50, bluespace = 50, bananium = 60)
```

Reply:
one static var persist across all objects of the same type, which means the game only needs the memory for one list instead of one for every lathe, good for memory optimization